release-team/role-handbooks/test-infra/README.md

```diff
@@ -1,23 +1,23 @@
 # Test Infra Lead Playbook
 
-## Overview of Test Infra Lead responsibilities
-
 Note: Currently, the test-infra lead has to be someone from Google GKE Engprod Team, in order to gain access to the prow cluster. This
 will change once we migrate our testing infrastructure under CNCF account. (xref kubernetes/test-infra#5085)
 
-There are two major area that test-infra lead need to take care during the release cycle, which are:
+There are three major area that test-infra lead need to take care during the release cycle, which are:
+
+1. [Create CI/Presubmit jobs for the new release, and populate the testgrid dashboard](#create-cipresubmit-jobs-for-the-new-release)
 
-1. Create CI/Presubmit jobs for the new release, and populate the testgrid dashboard
+1. [Configure merge automation for code slush, freeze, and thaw](#configure-merge-automation-for-code-slush-freeze-and-thaw)
 
-1. Watch for test infra status, make sure test infra is stable, react to test infra related issues and notify Release Lead and CI Signal Lead of issue status changes
+1. [Watch for test infra status, make sure test infra is stable, react to test infra related issues and notify Release Lead and CI Signal Lead of issue status changes](#ensure-the-stability-of-test-infra)
 
 It's important to ensure the stability of our test infra during the release cycle, so that we can get reliable testing signals throughout the release cycle.
 
 You can also work with @kubernetes/test-infra-maintainers or [test infra oncall](https://go.k8s.io/oncall) if you are blocked by anything.
 
 Also feel free to ping #sig-testing and #testing-ops to reach out for help.
 
-### Create CI/Presubmit jobs for the new release
+## Create CI/Presubmit jobs for the new release
```

Comment:
nit: can you please fix the typo 4th bullet below 
s/kuberketes_verify scenario/kubernetes_verify scenario